src/ServiceControlInstaller.Engine/Instances/ServiceControlBaseService.cs

```diff
@@ -202,8 +202,12 @@ protected string DefaultLogPath()
         {
             // The default Logging folder in ServiceControl uses the env vae"%LocalApplicationData%".  Since this is env user specific we'll determine it based on profile path instead.
             // This only works for a user that has already logged in, which is fine for existing instances
-            var userAccountName = UserAccount.ParseAccountName(Service.Account);
+            if (Service.Account == null)
+            {
+                return null;
+            }
 
+            var userAccountName = UserAccount.ParseAccountName(Service.Account);
```

Comment:
ParseAccountName explicitly handles `null` as input. It should convert null into the `LocalSystem` account.

Reply:
But I believe it throws an `ArgumentNullException` if the argument passed into the `Contains` call is null

Reply:
You're right - I just tested passing in null into `ParseAccountName` and that returned the system account.
So I think we can get rid of that null check.